examples/flatris/browserify/prepare-modules.js

```diff
@@ -2,7 +2,7 @@
 
 const prepareComponents = (modules) => {
   const components = {};
-  Object.keys(modules).forEach(componentName => {
+  Object.keys(modules).forEach((componentName) => {
```

Comment:
The rule is:
- If you write a function that takes one argument, and its body does not have curly braces, then do not apply parenthesis to the argument. If its body does have curly braces, then the argument must be surrounded by parenthesis.